src/System.CommandLine/ArgumentArity.cs

```diff
@@ -72,14 +85,29 @@ public ArgumentArity(int minimumNumberOfValues, int maximumNumberOfValues)
             return null;
         }
 
+        /// <summary>
+        /// Must not have any arguments.
+        /// </summary>
         public static IArgumentArity Zero => new ArgumentArity(0, 0);
 
+        /// <summary>
+        /// May have one argument, but no more than one.
```

Comment:
Since we have the properties `MinimumNumberOfValues` and `MaximumNumberOfValues`, these comments should probably refer to "values" rather than "arguments".

Reply:
Agreed. I've changed them all.